raiden/settings.py

```diff
@@ -46,6 +46,6 @@
 ETHERSCAN_API = 'https://{network}.etherscan.io/api?module=proxy&action={action}'
 
 RED_EYES_CONTRACT_VERSION = '0.4.0'
-DEVELOPMENT_CONTRACT_VERSION = '0.3._'
+DEVELOPMENT_CONTRACT_VERSION = '0.10.0'
```

Comment:
Shouldn't this be 0.14.0 ???

Reply:
The pypi package's version is different than the ones we have for contracts-version.
pypi version: 0.14.0
contracts version: 0.10.0

Here: https://github.com/raiden-network/raiden-contracts/blob/v0.14.0/raiden_contracts/data/source/raiden/TokenNetwork.sol#L14

Reply:
Duh ... :sweat_smile: 

I remember last week I was explaining that to someone and now even I got confused. We gotta do something about the drift between packages and contract version. If it confused us, imagine the users.

Reply:
Yeah. Maybe `contract_version` should just look very different.  https://github.com/raiden-network/raiden-contracts/issues/584